Scripts/AppCenterAnalytics/build-appcenteranalytics-tvos-framework.sh

```diff
@@ -41,7 +41,7 @@ cp -f "${SRCROOT}/${FMK_NAME}/Support/tvOS.modulemap" "${INSTALL_DIR}/Modules/mo
 # Copies the headers and resources files to the final product folder.
 cp -R "${SRCROOT}/${WRK_DIR}/Release-appletvos/include/${FMK_NAME}/" "${INSTALL_DIR}/Headers/"
 
-# Uses the Lipo Tool to merge both binary files (i386 + arm64) into one Universal final product.
+# Uses the Lipo Tool to merge both binary files (i386/x86_64 + armv7/armv7s/arm64) into one Universal final product.
```

Comment:
Is this a correct change?

Reply:
yes

Reply:
I would think so... It's just a comment, and wasn't the point of this to make a frankenbinary just like it's describing here?

Reply:
Oh I see, it's for tvOS... is tvOS available for arm64e?

Reply:
No, iOS only (see comment below).